.github/workflows/test.yml

```diff
@@ -6,15 +6,16 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.8, 3.9, '3.10', 3.11, '3.12-dev', pypy3.8, pypy3.9]
```

Comment:
Don't drop, please pypy3.8.

Reply:
https://www.pypy.org/download.html

Reply:
https://github.com/actions/setup-python/blob/main/docs/advanced-usage.md#available-versions-of-python-and-pypy